src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carCost = 40;
+  const mediumRent = 20;
+  const largeRent = 50;
+
+  if (days >= 7) {
+    return carCost * days - largeRent;
+  }
+
+  if (days >= 3) {
+    return carCost * days - mediumRent;
+  }
+
+  return carCost * days;
```

Comment:
you repeat `carCost * days` 3 times, you should create variable for this calculation